apps/analysis_framework/migrations/0041_widget_mapping.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 3.2.17 on 2024-02-22 05:02
+from django.db import migrations
+import copy
+
+
+def analysis_framework_widgets_mapping(apps, schema_editor):
+    AnalysisFramework = apps.get_model('analysis_framework', 'AnalysisFramework')
+    af_qs = AnalysisFramework.objects.filter(properties__isnull=False)
+    for af in af_qs:
+        if af.prop == {}:
+            continue
+        af_prop = af.properties
+        # Remove super legacy config
+        af_prop.pop('old_stats_config', None)
+
+        # For reassurance
+        af_prop = {
+            'old_stats_config': copy.deepcopy(af_prop),
+        }
+
+        # Migrate legacy config to latest
+        # -- Widget1D
+        af_prop['stats_config']['widget_1d'] = (
+            af_prop['stats_config'].get('widget_1d') or
+            af_prop['stats_config'].get('widget1d') or
+            []
+        )
+
+        # -- Widget2D
+        af_prop['stats_config']['widget_2d'] = (
+            af_prop['stats_config'].get('widget_2d') or
+            af_prop['stats_config'].get('widget2d') or
+            []
+        )
+
+        # -- Organigram
+        af_prop['stats_config']['organigram_widgets'] = (
+            af_prop['stats_config'].get('affected_groups_widgets') or
+            af_prop['stats_config'].get('organigram_widgets') or
+            af_prop['stats_config'].get('organigram_widget') or
+            []
+        )
+
+        # -- Multiselect
+        af_prop['stats_config']['multiselect_widgets'] = (
+            af_prop['stats_config'].get('specific_needs_groups_widgets') or
+            af_prop['stats_config'].get('multiselect_widgets') or
+            af_prop['stats_config'].get('multiselect_widget') or
+            []
+        )
```

Comment:
We need also need `demographic_groups_widget`
https://github.com/the-deep/server/pull/1412/files#diff-84ff5706f60a2d9653d58499e8713bb009e26611be9b4d41f5c4b82488980893L196-L209